res/translations/source_copy_allow_list.xml

```diff
@@ -0,0 +1,3755 @@
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Triplets</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Q</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Q 1</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Q 2</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Flanger</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Regen</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Mix</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Autopan</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Bessel4 ISO</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Res</source>
+    <allow_all_languages>false</allow_all_languages>
+    <allowed_languages>
+      <language>ca</language>
+      <language>de</language>
+      <language>el</language>
+      <language>en_CA</language>
+      <language>en_GB</language>
+      <language>es</language>
+      <language>es_419</language>
+      <language>es_AR</language>
+      <language>es_CO</language>
+      <language>es_CR</language>
+      <language>es_EC</language>
+      <language>es_ES</language>
+      <language>es_MX</language>
+      <language>es_PA</language>
+      <language>es_UY</language>
+      <language>fr_CA</language>
+      <language>fr_CI</language>
+      <language>hu</language>
+      <language>it</language>
+      <language>nl</language>
+      <language>pt</language>
+      <language>pt_BR</language>
+      <language>pt_PT</language>
+      <language>sl</language>
+      <language>sv</language>
+    </allowed_languages>
+  </message>
+  <message>
+    <source>%1:%2</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Bessel8 ISO</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>LR8 ISO</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>BQ EQ</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>BQ EQ/ISO</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Loudness</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>16 bits</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>24 bits</source>
+    <allow_all_languages>true</allow_all_languages>
+  </message>
+  <message>
+    <source>Param EQ</source>
+    <allow_all_languages>true</allow_all_languages>
```

Comment:
questionable

Reply:
What do you mean? Any suggestion?

Reply:
Well, in other languages, it may be translated differently. I don't think we can make this blanket statement that its "Param EQ" in all languages. This concern applies to many strings here.